### PR TITLE
Migrate nsproj to esproj

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**

### CODE REVIEW

File: docs/github.md [hunk 1]
```diff
@@ -1,34 +0,0 @@
-# GitHub Actions params fetching
```
Comment (pavel-mikula-sonarsource) [inline docs/github.md:1]: This is another forgotten cleanup. This file is now out of date after removal of v1

File: node_modules/.bin/prettier
Comment (sebastien-marichal) [inline node_modules/.bin/prettier:1]: I am surprised to see `node_modules` pushed into the repository.
Is it expected?

**Review by sebastien-marichal — APPROVED** [review]
LGTM!
I don't understand why the `node_modules` folder has been committed.
In any case, this is not related to this PR.

File: node_modules/.bin/prettier
Comment (pavel-mikula-sonarsource) [reply node_modules/.bin/prettier:1]: Yes, because there's no "npm install" at action runtime. All files that are needed to run needs to be in the repo

**Pull request merged**